src/main/java/net/logstash/logback/decorate/ReplaceCharsJsonFactoryDecorator.java

```diff
@@ -0,0 +1,131 @@
+package net.logstash.logback.decorate;
+
+import com.fasterxml.jackson.core.SerializableString;
+import com.fasterxml.jackson.core.io.CharacterEscapes;
+import com.fasterxml.jackson.core.io.SerializedString;
+import com.fasterxml.jackson.databind.MappingJsonFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class ReplaceCharsJsonFactoryDecorator implements JsonFactoryDecorator {
+
+    private static class CustomizableCharacterReplacer extends CharacterEscapes {
+
+        private final int[] escapeCodesForAscii;
+        private final List<Replace> replacements = new ArrayList<>();
+
+        private CustomizableCharacterReplacer() {
+            escapeCodesForAscii = new int[128];
+            Arrays.fill(escapeCodesForAscii, ESCAPE_NONE);
```

Comment:
I think this should start with `CharacterEscapes.standardAsciiEscapesForJSON()` by default rather than full of `ESCAPE_NONE`, mainly because the `standardAsciiEscapesForJSON()` are used by default if this `JsonFactoryDecorator` is not used.

i.e. If you happen to configure the `JsonFactoryDecorator` with no escapes, the behavior should be the same as if you didn't configure the `JsonFactoryDecorator` at all.